NCTRC/frontend/screens/CovidInformationScreen.tsx

```diff
@@ -15,126 +19,153 @@ interface CovidInformationScreenProps {
     navigation: CovidInformationScreenNavigationProps;
 }
 
-const styles = StyleSheet.create({});
-const maxCap = 10;
-const currentCap = 8;
 
 const CovidInformationScreen: React.FunctionComponent<CovidInformationScreenProps> = (props) => {
-    const { navigation, route } = props;
-    const { params } = route;
-    const { name, email } = params;
-    const [traveled, setTraveled] = React.useState(false);
-    const [concerns, setConcerns] = useState('');
-    const [contact, setContact] = React.useState(false);
-    const [gatherings, setGatherings] = React.useState(false);
-    const [isolate, setIsolate] = React.useState(false);
-    const [travelLabel, setravelLabel] = useState('');
-    const [contactLabel, setContactLabel] = useState('');
-    const [gatheringsLabel, setGatheringsLabel] = useState('');
-    const [isolateLabel, setIsolateLabel] = useState('');
+    let { navigation, route } = props;
+    let { params } = route;
+    let { firstName, lastName, email } = params;
+    let [traveled, setTraveled] = React.useState('');
+    let [concerns, setConcerns] = useState('');
+    let [symptoms, setSymptoms] = React.useState('');
+    let [gatherings, setGatherings] = React.useState('');
+    let [covidTest, setCovidTest] = React.useState('');
+    let [tempurature, setTempurature] = React.useState(0);
+    let yesQuestion = ""; 
 
     return (
-        <ScrollView>
-            <View>
-                <Image source={require('./../assets/NCTRClogo.png')} style={{ width: 10, height: 10 }}></Image>
-                <Text>Name</Text>
-                <TextInput placeholder={name} editable={false} />
-                <Text>Email</Text>
-                <TextInput placeholder={email} editable={false} />
-                <Text>
-                    Within the past 14 days, have you had contact with anyone that you know had COVID-19 or COVID-like
-                    symptoms?
+        <ScrollView style={styles.scrollContainer}>
+        <TouchableOpacity style={styles.backButton} onPress={() => navigation.pop()}>
+                 <Image source={require('./../assets/backbutton.png')} style={{ width: 75, height: 75 }}></Image>
+            </TouchableOpacity> 
+            <TouchableOpacity style={styles.logo} onPress={() => navigation.popToTop()}>
+                 <Image source={require('./../assets/NCTRClogo.png')} style={{ width: 150, height: 150 }}></Image>
```

Comment:
The questions should be saved in the config as an array that you loop through to generate JSX. This way it will be easy to add/remove questions.